### PR TITLE
First approach for load testing remotely

### PR DESCRIPTION
**Pull request by ausias-armesto**
Creates a new manual pipeline to execute the load testing from github actions

### CODE REVIEW

**Review by Teebor-Choka — APPROVED** [review]
@QYuQianchen

File: devkit/loadtesting-k6/package.json [hunk 1]
```diff
@@ -12,9 +12,13 @@
...
-        "start:discovery-platform": "k6 run src/discovery-platform/artificial.js --out json=results/results-discovery-platform.json"
+        "start:discovery-platform": "k6 run src/discovery-platform/artificial.js --out json=results/results-discovery-platform.json",
+        "docker:build": "cd ../../ ; docker build --platform linux/amd64 --tag europe-west6-docker.pkg.dev/rpch-375921/rpch/rpch-k6:latest -f devkit/loadtesting-k6/Dockerfile .",
+        "docker:push": "docker push europe-west6-docker.pkg.dev/rpch-375921/rpch/rpch-k6:latest"
     },
     "devDependencies": {
-        "@types/k6": "^0.47.1"
+        "@types/k6": "^0.47.1",
+        "@types/node": "^20.8.5",
+        "typescript": "^5.2.2"
```
Comment (esterlus) [inline devkit/loadtesting-k6/package.json:22]: Do you actually need this?
The typescript version comes from the root package.json as part of the monorepo dependencies.

Comment (ausias-armesto) [reply devkit/loadtesting-k6/package.json:22]: I couldn't make it compile de Dockerfile

Comment (esterlus) [reply devkit/loadtesting-k6/package.json:22]: Your approach tries to ignore the mono repo structure.

If you look into the other apps and packages, they all have Dockerfiles that you could use as a baseline.

**Pull request merged**